File: Easy/count-good-triplets.java
import java.lang.Math;

class Solution {        // 26ms
    public int countGoodTriplets(int[] arr, int a, int b, int c) {
        int[][] dynArr = new int[arr.length][arr.length];
        for(int i = 0; i < arr.length; i++){
            for(int j = i; j < arr.length; j++){
                dynArr[i][j] = Math.abs(arr[i] - arr[j]);
            }
        }

        int ret = 0;
        for(int i = 0; i < arr.length - 2; i++){
            for(int j = i + 1; j < arr.length - 1; j++){
                for(int k = j + 1; k < arr.length; k++){
                    if(dynArr[i][j] <= a && dynArr[j][k] <= b && dynArr[i][k] <= c)
                        ret++;
                }
            }
        }
        return ret;
    }
}

File: Easy/count-good-triplets.py
class Solution:     # 533ms
    def countGoodTriplets(self, arr: List[int], a: int, b: int, c: int) -> int:
        dynArr = []
        for i in range(len(arr) - 1):
            temp = [0] * len(arr)
            for j in range(i, len(arr)):
                temp[j] = abs(arr[i] - arr[j])
            dynArr.append(temp)
                
        ret = 0
        for i in range(len(arr) - 2):
            for j in range(i + 1, len(arr) - 1):
                for k in range(j + 1, len(arr)):
                    if dynArr[i][j] <= a and dynArr[j][k] <= b and dynArr[i][k] <= c:
                            ret += 1
        return ret